dali/operators/audio/nonsilence_op.h

```diff
@@ -27,6 +27,8 @@
 #include "dali/pipeline/data/views.h"
 #include "dali/pipeline/operator/operator.h"
 
+#define NONSILENCE_TYPES (uint8_t, int8_t, uint16_t, int16_t, uint32_t, int32_t, uint64_t, int64_t, float)  // NOLINT
```

Comment:
```suggestion
#define NONSILENCE_TYPES uint8_t, int8_t, uint16_t, int16_t, uint32_t, int32_t, uint64_t, int64_t, float  // NOLINT
```
with that, you'll be able to print the list of supported types, e.g.:
```C++
          make_string("Unsupported input type: ", in.type(), "\nSupported types are : ",
                      ListTypeNames<NONSILENCE_TYPES>()));
```